infra/main.bicep

```diff
@@ -258,7 +311,7 @@ module openai 'core/ai/cognitiveservices.bicep' = {
         }
         sku: {
           name: 'Standard'
-          capacity: 30
+          capacity: azureOpenAIModelCapacity
```

Comment:
Linking #635

Reply:
I'll add it to the param file so we can close this off while we're here.